.gitignore

```diff
@@ -21,7 +21,7 @@ coverage/
 public/dist/
 uploads
 modules/users/client/img/profile/uploads
-config/env/local.js
+config/env/local*.js
```

Comment:
Ah. I thought that it would stop tracking changes to it but leave it in there. But if it will work then nevermind. :)